backend/apps/volontulo/serializers.py

```diff
@@ -26,6 +26,7 @@ class Meta:
             'name',
             'slug',
             'url',
+            'address',
```

Comment:
we alread have `address` here (see the first position in this tuple)

Reply:
Ok, address was deleted